docs/content/development/extensions-core/kinesis-ingestion.md

```diff
@@ -126,37 +126,37 @@ A sample supervisor spec is shown below:
+| `handoffConditionTimeout`             | Long           | Milliseconds to wait for segment handoff. It must be >= 0, where 0 means to wait forever.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | no (default == 0)                                                                                            |
+| `resetOffsetAutomatically`            | Boolean        | This option _MUST_ not be set if you don't want to ignore missing data from Kinesis. Once this is set, the supervisor _automatically_ resets the consumer offset for each Kinesis partitions whenever the next offset it's trying to fetch is less than the earliest available offset in Kinesis. The consumer offset will be reset to either the earliest or latest offset available in Kinesis depending on `useEarliestOffset` property of `KinesisSupervisorIOConfig`. Please note that this could lead _data loss_ (when `useEarliestOffset` = false) or _data duplicate_ (when `useEarliestOffset` = true). The missing offsets in Kinesis typically occurs when messages in Kinesis are no longer available for consumption and therefore won’t be ingested into Druid. If set to false then ingestion for that particular partition will halt and manual intervention is required to correct the situation, please see [Reset Supervisor API](../../operations/api-reference.html#supervisors). | no (default == false)                                                                                        |
```

Comment:
Typos:
- for each Kinesis partitions -> for each Kinesis partition
- depending on `useEarliestOffset` -> depending on the `useEarliestOffset`
- could lead _data loss_ -> could lead to _data loss_
- missing offsets in Kinesis typically occurs -> missing offsets in Kinesis typically occur